docs/README_ko.md

```diff
@@ -0,0 +1,79 @@
+<h1 align="">Indonesian Area API (<i>API Wilayah Indonesia</i>)</h1>
+
+<p>
+  <a href="https://nestjs.com"><img alt="NestJS" src="https://img.shields.io/badge/-NestJS-ea2845?style=flat-square&logo=nestjs&logoColor=white" /></a>
+  <a href="https://www.typescriptlang.org"><img alt="TypeScript" src="https://img.shields.io/badge/-TypeScript-007ACC?style=flat-square&logo=typescript&logoColor=white" /></a>
+  <a href="https://www.prisma.io"><img alt="Prisma" src="https://img.shields.io/badge/-Prisma-1B222D?style=flat-square&logo=prisma&logoColor=white" /></a>
+  <a href="https://www.mongodb.com"><img alt="MongoDB" src="https://img.shields.io/badge/-MongoDB-47A248?style=flat-square&logo=mongodb&logoColor=white" /></a>
+  <a href="https://www.postgresql.org"><img alt="PostgreSQL" src="https://img.shields.io/badge/-PostgreSQL-657991?style=flat-square&logo=postgresql&logoColor=white" /></a>
+  <a href="https://www.mysql.com"><img alt="MySQL" src="https://img.shields.io/badge/-MySQL-00688F?style=flat-square&logo=mysql&logoColor=white" /></a>
+</p>
+
+**English** | [हिन्दी](docs/README_hi.md) | [Tagalog](docs/README_tl.md) | [한국어](docs/README_ko.md)
```

Comment:
The link to the other file is incorrect. Use relative path (see the [following article](https://www.redhat.com/sysadmin/linux-path-absolute-relative)).

```suggestion
[English](../README.md) | [हिन्दी](README_hi.md) | [Tagalog](README_tl.md) | **한국어**
```

Please also adjust if there are other links that go to other files.

Reply:
Ah I understand! Thanks.

Reply:
Have you checked the other links? Such as links to installation.md and CONTRIBUTING.md files.
I still find that links don't work.

Reply:
@fityannugroho Sorry for that. I think every link will work well.
Let me know after checking!